migrate.py

```diff
@@ -1851,19 +1880,21 @@ def rewrite_integration_tests(test_dirs, checkout_dir, collection_dir, namespace
                 if ext in BAD_EXT:
                     continue
                 elif ext in ('.py',):
-                    import_deps, docs_deps = rewrite_py(src, dest, collection, spec, namespace, args)
+                    import_deps, docs_deps = rewrite_py(src, dest, collection, spec, namespace, args, cdm=cdm)
 
                     for dep_ns, dep_coll in import_deps + docs_deps:
-                        integration_tests_add_to_deps((namespace, collection), (dep_ns, dep_coll))
+                        if cdm:
+                            cdm.add_integration_dep('.'.join([dep_ns, dep_coll]), dest)
```

Comment:
just use an f-string
```suggestion
                            cdm.add_integration_dep(f'{dep_ns}.{dep_coll}', dest)
```